src/store/index.js

```diff
@@ -0,0 +1,120 @@
+import Vuex from 'vuex'
+import Vue from 'vue'
+import firebase from 'firebase'
+import db from '../main.js'
+Vue.use(Vuex)
+
+export default new Vuex.Store({
+  state: {
+    loginUserName: '',
+    loginUserMoney: '',
+    loginUserID: '',
+    userStorage: '',
+    targetUserID: '',
+    targetUserMoney: '',
+    firstMoney: 1000, //新規ユーザーの最初の所持金額
+  },
+  actions: {
+    userJoinAction({ commit }, { email, password, name }) {
+      const userJoinInformation = {
+        mailInformation: email,
+        passwordInformation: password,
+        nameInformation: name,
+      }
+      commit('userJoinAction', userJoinInformation)
+    },
+    userLoginAction({ commit }, { email, password }) {
+      const loginInformation = {
+        loginMailInformation: email,
+        loginPasswordInformation: password,
+      }
+      commit('userLoginAction', loginInformation)
+    },
+    userLogoutAction(state) {
+      state.loginUserName = ''
+      state.userStorage = ''
+      state.loginUserMoney = ''
+      firebase.auth().signOut()
+    },
+    transferMoneyAction({ commit }, { name, money }) {
+      const transferInformation = {
+        transferNameInformation: name,
```

Comment:
プロパティ名もnameとしてしまえば、以下のように省略記法が使用できます。
```suggestion
      const transferInformation = { name }
```